### PR TITLE
Feature/better error handling

### PR DESCRIPTION
**Pull request by avnersorek**
prerequisite for https://github.com/dorbel-tech/dorbel-app/issues/364

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,12 +17,12 @@
     "auth0": "^2.5.0",
     "bluebird": "^3.4.6",
     "bunyan": "~1.8.1",
-    "bunyan-sumologic": "^1.0.1",
+    "bunyan-sumologic": "avnersorek/bunyan-sumologic",
```
Comment (virtser) [inline package.json]: fork into dorbel-tech?

File: src/utils/serverRunner.js [hunk 2]
```diff
@@ -1,27 +1,44 @@
...
 const co = require('co');
 const throng = require('throng');
-const gracefulShutdown = require('http-graceful-shutdown');
+const gracefulShutdown = require('./gracefulShutdown');
 
-const logger = require('../logger').getLogger(module);
-const isDevelopment = process.env.NODE_ENV === 'development';
+const Logger = require('../logger');
+const logger = Logger.getLogger(module);
+const isDevelopment = (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test');
```
Comment (virtser) [inline src/utils/serverRunner.js]: Why on test?

Comment (avnersorek) [inline src/utils/serverRunner.js]: For the tests ;)
Running a the cluster in tests wasn't simple so the units tests are testing this without the cluster part.

File: package.json (hunk 1 above)
Comment (avnersorek) [inline package.json]: I hope this gets merged in to the main repo and then I'll delete it and we can get it from NPM

**Pull request merged**